src/lib/filters.js

```diff
@@ -9,7 +9,7 @@ import he from 'he'
  * @param {Int} height
  */
 export function thumbnail (relativeUrl, width, height) {
-  return config.images.baseUrl + parseInt(width) + '/' + parseInt(height) + '/resize' + relativeUrl
+  if (relativeUrl) return config.images.baseUrl + parseInt(width) + '/' + parseInt(height) + '/resize' + relativeUrl
 }
```

Comment:
And what if not? :) it should return sth - for example placeholder URL from the config or just plain string, but please not nothing; it violates LSP (Liskov Substitiution Principle)